events/events-team/comms/README.md

```diff
@@ -1,55 +1,191 @@
-# Communications Lead Handbook
+# Communications Team Handbook
 
 This document defines communications activities needed to run a
-Contributor Summit.
+Kubernetes Contributor Summit (KCS). The Comms Team consists of 1-2 lead(s), shadows, volunteers, at least 1 emeritus lead, and a Website Manager.
 
-- [Overview](#overview)
-- [Skills and Qualifications](#skills-and-qualifications)
-- [Responsibilities](#responsibilities)
-  - [Communication Channels and Property Access](#communication-channels-and-property-access)
+Comms Leads are responsible for leading the comms team, training shadows, and ensuring all comms activities are planned, prioritized, and completed successfully. They are committed to run the comms team for the current cycle and advise as Emeritus KCS Comms Lead in the following comms cycle (2 cycles, 1 year). Must have [Kubernetes Org Membership]. Expected to attend in-person.
+
+Shadows will learn both how to do comms and how to run the KCS Comms team, as directed by the Comms Lead. They are committing to be involved with the event team for 1.5 years or 3 comms cycles (shadow, lead, and emeritus). Must either have [Kubernetes Org Membership] or be planning to pursue org membership.
+
+Volunteers are expected to assist with comms during the current KCS cycle, as directed by the lead. Volunteers do not need to have or be pursuing [Kubernetes Org Membership].
```

Comment:
Volunteers are GREAT!

But, for KCS Comms is it okay to bring in someone completely new to the project to help? What would that work look like? I think answering these two questions will need to be in the doc (an FAQ or something like that is an option too).

Reply:
Sure, why not?  It's a pretty good first contribution task.

Reply:
+1. One problem with comms is just the sheer volume. If someone wanted to pop in to help with getting some drafts written, but didn't want to commit to being on a leadership path, I would still welcome the help.